new.go

```diff
@@ -313,3 +326,26 @@ func newBuilder(ctx context.Context, store storage.Store, options BuilderOptions
 
 	return builder, nil
 }
+
+// getNetworkInterface creates the network interface
+func getNetworkInterface(store storage.Store, cniConfDir, cniPluginPath string) (nettypes.ContainerNetwork, error) {
+	conf, err := config.Default()
+	if err != nil {
+		return nil, err
+	}
+	// copy the config to not modify the default by accident
+	newconf := *conf
```

Comment:
This step shouldn't be necessary - the value we got back was created fresh.

Reply:
config.Default() returns a pointer to the global Config, so if I do not copy this it would change the global config which I do not want

Reply:
Oof, you're right, but that seems like a bug waiting to happen.